Makefile.proto

```diff
@@ -0,0 +1,71 @@
+# Copyright The OpenTelemetry Authors  -*- mode: makefile; -*-
```

Comment:
Is this intentional?

```suggestion
# Copyright The OpenTelemetry Authors
```

Or maybe?

```suggestion
#   -*- mode: makefile; -*-
# Copyright The OpenTelemetry Authors
```

Reply:
It is intentional - emacs recognizes `Makefile` or `makefile` as filenames to automatically enter Makefile mode, but not something that ends in `.proto`. The file-local variables can be anywhere on the first line.. so I guess it should probably not be confused with the Copyright notice.

But I'm also open to just removing it and remembering to go into Makefile mode when I open Makefile.proto.

Reply:
Ah, gotcha. I think the accepted change sounds appropriate than.